silkworm/silkrpc/http/request_handler.hpp

```diff
@@ -77,6 +80,8 @@ class RequestHandler {
     const commands::RpcApiTable& rpc_api_table_;
 
     const std::optional<std::string> jwt_secret_;
+
+    std::optional<std::vector<std::string>> allowed_origins_;
```

Comment:
It seems not necessary to make this field optional because I don't see any difference between `absence of value` (i.e. no CORS enabled) and `empty allowed origins` (i.e. no allowed origins, which again means no CORS). So, I suggest changing this to `const std::vector<std::string>&`.